pizzeria-backend/Controllers/ProductsController.cs

```diff
@@ -37,7 +39,14 @@ public async Task<IActionResult> GetProduct(int id)
             {
                 return NotFound("Product not found");
             }
-            return Ok(pr);
+            var prString = JsonConvert.SerializeObject(pr, new JsonSerializerSettings
```

Comment:
If you really need this there's probably a way to make it the default in Program.cs rather than copy pasting it everywhere. If not you can create a helper class or even you can extract this as an extension method to the object class (I can show you how to do that but let's leave it as a last resort if extracting in Program.cs doesn't work)

Reply:
Done(the Program.cs solution)